templates/frontends-pipeline-pr-checks/cypress_component_tests.sh

```diff
@@ -9,14 +9,12 @@ docker run --name $TEST_CONT -d -i --rm  "${CYPRESS_TEST_IMAGE}" /bin/bash
 docker cp -a . "${TEST_CONT}:/e2e/"
 
 docker exec -i $TEST_CONT sh -c "npm i"
-docker exec -i $TEST_CONT sh -c "npm run test:ct"
+docker exec -i $TEST_CONT sh -c "npm run ci:cypress-component-tests"
 
 RESULT=$?
 
+docker rm -f $TEST_CONT
```

Comment:
I think you only need to stop the containers, having created them using `--rm` flag they'll auto-delete on stop.